ui/assets/css/_header.scss

```diff
@@ -22,3 +21,14 @@
     padding-left: 0.2rem;
   }
 }
+
+@media (max-width: 52em) {
```

Comment:
Small detail, but would it make sense to swap these out with the [variables](https://github.com/basscss/basscss-sass/blob/48212a612efaa913599bbd273724622a00e0a7f4/scss/_media-object.scss#L9-L11) from basscss-sass? That way we could swap out of break points in one fell swoop.